handbook/marketing/adding_screenshots_screen_recording.md

```diff
@@ -0,0 +1,116 @@
+# Adding a screenshot or screen recording (GIF and video)
+
+> Note: This guide presumes macOS
+
+## TL;DR
+
+- Favor capturing only the part of the screen that’s required
+- Keep GIFs 15 seconds or less
+- GIFs are typically great for short "check this out" recordings, where videos are better at "how-to" or promotional style content
+- Keep GIF and images as small as possible with no retina sizes (just 1 x)
```

Comment:
How do you know if it is retina sized? Is there a setting to look for or way to check for this?

Reply:
Added explanation "(ratio of image size to intended display size is 1:1)"